pkgs/development/python-modules/flask-security/default.nix

```
@@ -111,15 +111,15 @@ buildPythonPackage rec {
       mongoengine
       mongomock
       peewee
-      pony
       pytestCheckHook
       requests
       zxcvbn
     ]
     ++ optional-dependencies.babel
     ++ optional-dependencies.common
     ++ optional-dependencies.fsqla
-    ++ optional-dependencies.mfa;
+    ++ optional-dependencies.mfa
+    ++ lib.optionals (pythonOlder "3.12") [ pony ];
```

https://github.com/pallets-eco/flask-security/blob/e386da617ffff93af5b3f41480d4a896468cf3e5/pyproject.toml#L78 says
```suggestion
    ++ lib.optionals (pythonOlder "3.11") [ pony ];
```
Can we just drop it completely?

---

only if we don't want to build it for python 3.11 anymore